source/install/self-managed-changelog.md

```diff
@@ -6,12 +6,107 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
+
+### Compatibility
+ - Updated Chromium minimum supported version to 116+.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a **Cancel** button to the **Delete category** modal.
+ - Added the ability to resize the channel sidebar and right-hand sidebar.
+ - Added two new filtering options (show all channel types, show private channels) to the **Browse channels** modal.
+ - Pre-packaged GitLab plugin version v1.7.0.
+ - Added additional reaction options when viewing threads or messages when the sidebar is larger than its minimum width.
+ - Added block changes to name, display name, and purpose for direct and group messages.
+ - Added a link to [notification documentation](https://docs.mattermost.com/preferences/manage-your-notifications.html) in the **Notification Settings** modal.
+ - Updated the post textbox measurement code to be more reliable.
+ - Pre-packaged Calls version v0.19.1.
+ - The ``invite`` slash command now supports custom user groups.
+ - Re-enabled the remote marketplace functionality, when configured as per ``PluginSettings.EnableRemoteMarketplace`` [documentation](https://docs.mattermost.com/configure/plugins-configuration-settings.html#plugins-enableremotemarketplace).
+ - Added the ability to convert a group message channel to a private channel.
+ - Group message channels now behave like direct message channels, showing a new mention for every new message.
+
+#### Administration
+ - Added ``mattermost-plugin-api`` into the mono repo.
+ - Updated the public server module version to v0.0.9.
+ - Added 2 new URL parameters to ``GET /api/v4/groups``: ``include_archived`` and ``filter_archived``. Added the ability to restore archived groups from the user groups modal.
+ - Added file storage information to the support package.
+ - A ``user_id`` is now included in all HTTP logs (debug level) to help determine who is generating unexpected traffic.
+ - Added new URL Parameter to ``GET /api/v4/groups`` and ``GET /api/v4/groups/:group_id``. ``include_member_ids`` will add all the members ``user_ids`` to the group response objects. You can now also add group members to a channel, any members that are not part of the team can be added to the team through this flow and subsequently added the channel.
+
+#### Plugin Changes
+ - Added new frontend plugin extension point for the new messages separator bar.
+ - Added a new plugin extensibility point to add actions to the code blocks.
+ - Added the plugin hook ``UserHasBeenDeactivated``.
+ - Added a new server side plugin API method to set the searchable content for a file info (``SetFileSearchableContent``). The ``MessageHasBeenPosted`` plugin hook is now executed after the attachments are linked to the post.
+
+### Bug Fixes
+ - Fixed keyboard support for the left-hand side channel menu, the left-hand side category menu, and the post dot menu.
+ - Fixed display name in the ``comment_on`` component.
+ - Fixed an issue with keyboard support for some menus with submenus.
```

Comment:
Does this bullet point overlap with the first bullet point in this Bug Fixes section?